cloudimagemetadata.go

```diff
@@ -211,6 +229,7 @@ func importCloudImageMetadataV1(source map[string]interface{}) (*cloudimagemetad
 		Source_:          valid["source"].(string),
 		Priority_:        int(valid["priority"].(int64)),
 		ImageId_:         valid["image-id"].(string),
+		ExpireAt_:        expireAtPtr,
```

Comment:
Do we still want to populate it if it does not exist?

Reply:
It will be nil, same as for how root storage size is handled if it is missing.